doc/devdocs/modules/launcher/plugins/registry.md

```diff
@@ -0,0 +1,96 @@
+# Registry Plugin
+
+The registry plugin allow the users to search into the Windows registry.
+
+## Special functions (differ from the regular functions)
+
+* Support full base keys and short base keys (e.g. `HKLM` for `HKEY_LOCALE_MACHINE`).
+* Show count of sub-keys and count of values in the second result line.
+
+## The Windows Registry
+
+The registry contains all settings for the Windows operating system and many settings of the installed (Windows only) programs.
+
+*Note: Linux and macOS program ports typical store the settings in it's own configuration files and not in the Windows registry.*
+
+For more information about the Windows registry, see [the official documentation about it](https://docs.microsoft.com/en-us/windows/win32/sysinfo/registry).
```

Comment:
nit: "the official documentation about it" seems too verbose. I think "the official documentation" is fine